values.yaml

```diff
@@ -387,12 +437,22 @@ topologySpreadConstraints: []
 resources:
   limits:
     memory: 2Gi
+    ephemeral-storage: 1Mi
```

Comment:
This came from kube-score, it complains that both limits and requests should have some value for the ephemeral storage... WDYT @yaskoo

Reply:
Decided to remove them for now.